specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/analyzeconversations.json

```
@@ -1928,7 +1928,7 @@
         },
         "redactionSource": {
           "type": "string",
-          "description": "For transcript conversations, this parameter provides information regarding which content type (ITN, Text, Lexical, Masked ITN) should be used for redaction. The entities detected the service in will correspond to the text type selected here.",
+          "description": "For transcript conversations, this parameter provides information regarding which content type (ITN, Text, Lexical, Masked ITN) should be used for entity detection. The details of the entities detected like the offset, length and the text itself, the service in will correspond to the text type selected here.",
```

Doesn't make sense as written. Perhaps this works?
```suggestion
          "description": "For transcript conversations, this parameter provides information regarding which content type (ITN, Text, Lexical, Masked ITN) should be used for entity detection. The details of the entities detected - like the offset, length and the text itself - will correspond to the text type selected here.",
```